server/cmwell-sparql-agent/src/main/resources/reference.conf

```diff
@@ -10,5 +10,8 @@ cmwell.agents {
     init-delay = 10 seconds
     max-delay = 5 minutes
     config-polling-interval = 30 seconds
+    consumer {
```

Comment:
By manually adding this config to the STP agent, the STP  stand alone won't have the ability to change the length-hint.

You should:
Either
* add this feature the the stand alone STP 
* or revert back to the old commit and get the config from the application.conf of STP instead of the generic reference.con. This way you won't need to pass the parameter all along the way, just change the application.conf of the stand alone and the STP agent. It's your call but I prefer it this way - much cleaner and we will have this ability also for the consumer.

Reply:
@YoniMataraso - do we also want the stand alone to change length hint. I didn't think it was the case. If we decide I will have to change a few things in the PR

Reply:
Will create stand alone data tools changes in a future PR after discussion with @YoniMataraso